ammo-fe/src/redux/reducers/webSocketReducer.ts

```diff
@@ -0,0 +1,27 @@
+import { Bullet } from 'shared/typings/Bullet';
+
+type WebSocketReducerAction = {
+    type: 'connected' | 'disconnect' | 'receivedBullet';
```

Comment:
Usually redux actions are full uppercased. Also, be consistant : it's either 'connected' & 'disconnected' or 'connect' & 'disconnect'. The first one is better imo because they are adjectives and not verbs (verbs suggest methods)